include/mbedtls/rsa_internal.h

```diff
@@ -161,11 +161,12 @@ int mbedtls_rsa_deduce_crt( const mbedtls_mpi *P, const mbedtls_mpi *Q,
  * \return
  *                 - 0 if the following conditions are satisfied
  *                   if all relevant parameters are provided:
- *                    - P prime if f_rng != NULL
- *                    - Q prime if f_rng != NULL
- *                    - 1 < N = PQ
+ *                    - P prime if f_rng != NULL (%)
+ *                    - Q prime if f_rng != NULL (%)
+ *                    - 1 < N = P * Q
  *                    - 1 < D, E < N
  *                    - D and E are modular inverses modulo P-1 and Q-1
+ *                   (%) This is only done if MBEDTLS_GENPRIME is defined.
```

Comment:
The footnote actually appears on the same line as “D and E …”. To put it on a paragraph of its own, you need to move it below the bullet list, with a blank line above.

Not that it matters since we exclude `*_internal.h` in the doxygen configuration.